routers/api/v1/repo/issue.go

```diff
@@ -165,14 +170,32 @@ func SearchIssues(ctx *context.APIContext) {
 			PriorityRepoID:     ctx.QueryInt64("priority_repo_id"),
 			IsPull:             isPull,
 		})
-	}
+		if err != nil {
+			ctx.Error(http.StatusInternalServerError, "Issues", err)
+			return
+		}
 
-	if err != nil {
-		ctx.Error(http.StatusInternalServerError, "Issues", err)
-		return
+		filteredCount, err = models.CountIssues(&models.IssuesOptions{
+			ListOptions: models.ListOptions{
+				Page:     ctx.QueryInt("page"),
+				PageSize: issueCount,
```

Comment:
```suggestion
				PageSize: setting.UI.IssuePagingNum,
```

Reply:
☝️ if you dedub things this is not needed anyway ...

Reply:
This is for the purpose of getting the maximum value that matches the filter.
Therefore, the current maximum value is set as an option.